tests/kernel/sleep/src/main.c

```diff
@@ -101,16 +102,12 @@ static void test_thread(int arg1, int arg2)
 	align_to_tick_boundary();
 
 	start_tick = k_uptime_get_32();
-
-	/* FIXME: one tick less to account for
-	 * one  extra tick for _TICK_ALIGN in k_sleep
-	 */
-	k_sleep(ONE_SECOND - TICKS_PER_MS);
+	k_sleep(ONE_SECOND);
 	end_tick = k_uptime_get_32();
 
-	if (!sleep_time_valid(start_tick, end_tick, ONE_SECOND)) {
+	if (!sleep_time_valid(start_tick, end_tick, ONE_SECOND_ALIGNED)) {
```

Comment:
Again, not relevant to this review, but that kind of logic is racy in general.  Probably fine in this one test, but...